Core/App.Web/Builders/WebApplicationSetupBuilder.cs

```diff
@@ -12,6 +12,8 @@ internal class WebApplicationSetupBuilder : ApplicationSetupBuilder, IWebApplica
     public IControllerOptions Controller { get => ControllerOptions; }
     internal ControllerOptions ControllerOptions { get; } = new ControllerOptions();
 
+    public ICollection<Type> RequestPipelineFilterMetadata { get; } = new List<Type>();
```

Comment:
Maybe move this to the ControllerOptions and use a method AddFilters(params IFilterMetadata[] filters). This way multiple libs can start adding filters. To should then be gathered in an internal List<IFilterMetadata>. Then in StartupBase ln 82, you use that internal list to actually add those filters to the controller options. Would that make sense?

Reply:
Make sense, I'll update the code

Reply:
Code updated